python/ql/test/experimental/dataflow/regression/pointsto.ql

```diff
@@ -0,0 +1,28 @@
+private import python
+import experimental.dataflow.DataFlow
+
+predicate pointsToOrigin(DataFlow::DataFlowCfgNode pointer, DataFlow::DataFlowCfgNode origin) {
+  origin = pointer.pointsTo().getOrigin()
+}
+
+class PointsToConfiguration extends DataFlow::Configuration {
```

Comment:
Maybe add a short comment somewhere that describes what this "query" does?

Reply:
Done, thanks.